Jenkinsfile

```diff
@@ -145,6 +161,44 @@ pipeline {
                 }
             }
         }
+        stage('Verify changes') {
+            agent any
+            steps{
+                script {         
+
+                    retry(3) { checkout scm }
+                    sh "git clean -xffd"
+
+                    def commitHash = sh(script: "git rev-parse HEAD | tr '\\n' ' '", returnStdout: true)
+                    sh(script: "git pull && git checkout develop", returnStdout: false)
```

Comment:
Can Jenkins figure out to which branch a PR is done to?

I tried to run a PR to a non-develop branch and that fails. See https://jenkins.mc-stan.org/blue/organizations/jenkins/Math%20Pipeline/detail/PR-1725/2/pipeline/

Reply:
If this is not doable or hard to do, I am also fine with skipping tests on PRs not done to develop. They will have to be done to develop eventually anyways.

Reply:
Hey, based on the [environment variables](https://ci.eclipse.org/webtools/env-vars.html/) we can use `CHANGE_TARGET` here, I've pushed a change now.

Reply:
Great! This all works now https://jenkins.mc-stan.org/blue/organizations/jenkins/Math%20Pipeline/detail/PR-1725/14/pipeline

Reply:
sweet!